action.yml

```diff
@@ -18,13 +18,15 @@ inputs:
   stale-issue-label:
     description: 'The label to apply when an issue is stale'
     default: 'Stale'
-  exempt-issue-label:
-    description: 'The label to apply when an issue is exempt from being marked stale'
+  exempt-issue-labels:
+    description: 'The labels to use to exempt an issue from being marked stale'
+    default: []
```

Comment:
Is this valid for the action.yml file? Seems to be valid Yaml, but not sure if it does what I expect it to (result in an empty array if no argument is provided in the `with` block).

Reply:
Sure, looks valid: https://nodeca.github.io/js-yaml/#yaml=ICBleGVtcHQtaXNzdWUtbGFiZWxzOgogICAgZGVzY3JpcHRpb246ICdUaGUgbGFiZWxzIHRvIHVzZSB0byBleGVtcHQgYW4gaXNzdWUgZnJvbSBiZWluZyBtYXJrZWQgc3RhbGUnCiAgICBkZWZhdWx0OiBbXQ==